src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -1,5 +1,63 @@
 {{-- Date Range Backpack CRUD filter --}}
 
+@php
+    $filterOptions = $filter->options['date_range_options'] ?? [];
+    $filterOptions['locale'] = $filter->options['date_range_options']['locale'] ?? [];
+
+
+
+
+    //initialize bare default configurations
```

Comment:
@pxpm I see this has been created before we settled on a readability standard for this kind of things. Like we did in fields, I think this would be much easier to read if we created a configuration array, then used ```array_merge()``` to merge the two.

Right now this looks very intimidating 😄 

I've created a different PR which works fine, as an example of how I see this part of the field becoming - https://github.com/Laravel-Backpack/CRUD/pull/2500/files